yarn-project/pxe/src/note_processor/utils/add_public_values_to_payload.ts

```diff
@@ -5,16 +5,16 @@ import { ContractNotFoundError } from '@aztec/simulator';
 import { type PxeDatabase } from '../../database/pxe_database.js';
 
 /**
- * Inserts publicly delivered nullable fields into the note payload.
+ * Inserts publicly delivered values into the note payload.
```

Comment:
Did this naming change here as I consider nullable to be the note struct members which can be delivered in public but might not while the public values are the members which go actually delivered in public.